module_checksum_upgrade/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Module Checksum Upgrade',
+    'summary': 'Base mechanism to detect changes to '
+               'installed Odoo modules and upgrade them '
+               'automatically',
+    'version': '9.0.1.0.0',
+    'category': 'Extra Tools',
+    'website': 'https://github.com/OCA/server-tools/',
+    'author': 'ACSONE SA/NV, '
+              'LasLabs, '
+              'Juan José Scarafía, '
```

Comment:
This should Ingenería ADHOC I think cc @jjscarafia

Reply:
I don't know, I copied this from module_auto_update.